cep-purls.md

```diff
@@ -0,0 +1,100 @@
+<table>
+<tr><td> Title </td><td> Add package-urls to PackageRecord </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Bas Zalmstra &lt;bas@prefix.dev&gt;</td></tr>
+<tr><td> Created </td><td> Nov 23, 2023</td></tr>
+<tr><td> Updated </td><td> Nov 23, 2023</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CEP describes a change to the `PackageRecord` format and the corresponding `repodata.json` file to include `purls` (Package Urls) of repackaged packages to identify packages across multiple ecosystems.
+
+## Specification
+
+We propose to add the optional `purls: [string]` field to `PackageRecord`.
```

Comment:
As @jaimergp mentioned, we should also mention in this CEP where in the built conda package this should live and where it is specifiable in the recipe.
IMO, `index.json` is a good fit for it as IMO, it should also go into repodata (at least into sharded).

Adding it to `index.json` will lead to `conda-index` automatically adding it to `repodata.json`. This is IMO a bad default in `conda-index`; it should instead just keep a whitelist of things to put into `repodata.json` instead of putting everything from `index.json` in there.